js/Review/index.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * WordPress dependencies.
+ */
+import {
+    __,
+    sprintf
+} from '@wordpress/i18n';
+
+import apiFetch from '@wordpress/api-fetch';
+
+import domReady from '@wordpress/dom-ready';
+
+import {
+    Button,
+    Modal
+} from '@wordpress/components';
+
+import {
+    createRoot,
+    useState
+} from '@wordpress/element';
+
+const App = () => {
+    const [ isOpen, setOpen ] = useState( true );
+
+    const closeModal = async () => {
+        console.log( 'closeModal' );
+        try {
+            await apiFetch({
+                path: '/wp/v2/settings',
+                method: 'POST',
+                data: {
+                    feedzy_review_notice: 'yes',
+                },
+            });
+        } catch ( error ) {
+            console.error( __( 'Error updating setting:', 'feedzy-rss-feeds' ), error );
+        } finally {
+            setOpen( false );
+        }
+    };
+
+    if ( ! isOpen ) {
+        return null;
+    }
+
+    return (
+        <Modal
+            title={ __( 'Congrats, You\'ve Reached an Impressive Milestone! 🎉', 'feedzy-rss-feeds' ) }
+            size="medium"
+            shouldCloseOnClickOutside={ false }
+            onRequestClose={ closeModal }
+        >
+            <p
+                dangerouslySetInnerHTML={{
+                    __html: sprintf(
+                    // translators: %1$s is an opening strong tag, %2$s is a closing strong tag.
+                    __(
+                        "You've successfully imported %1$s more than 100 posts %2$s using Feedzy!",
```

Comment:
You can also give 100 as `%3$s`. If we change the number, we must translate the string again.